conversant/demo/streamlit_example.py

```diff
@@ -26,6 +26,7 @@
 def get_reply() -> None:
     """Replies query from the message input, and resets the message input"""
     _ = st.session_state.bot.reply(query=st.session_state.message_input)
+    # Reset user input
```

Comment:
I think this comment is a remnant from a merge conflict - sorry for the rapidly evolving state of the repo causing so many conflicts :( I want to acknowledge that it is a huge pain!

Reply:
No worries, I'm always solving the conflicts to prevent them from accumulating and be harder to fix